packages/skygear-core/src/types.ts

```diff
@@ -702,6 +702,7 @@ export interface _OIDCConfiguration {
   claims_supported: string;
   code_challenge_methods_supported: string;
   revocation_endpoint: string;
+  end_session_endpoint: string;
```

Comment:
Some fields here have wrong types. `scopes_supported`, `response_types_supported`, `grant_types_supported`, `subject_types_supported`, `id_token_signing_alg_values_supported`, `claims_supported`, `code_challenge_methods_supported`. I think we can just remove the fields we do not use in the SDK. That being said, just keep `*_endpoint` fields.